lib/front_desk.rb

```diff
@@ -0,0 +1,187 @@
+require_relative "reservation"
+require_relative "block_room"
+
+module Hotel
+  class FrontDesk
+    # attr_reader :all_rooms, :reservations_record, :block_reservations_only, :reserved_rooms_in_blocks
+    attr_accessor :all_rooms, :reservations_record, :block_reservations_only, :reserved_rooms_in_blocks
```

Comment:
Why did you choose to make this `accessor` instead of `reader`? Your tests still run the same way if you make it `reader`, so I would choose this over `accessor`